ui/src/components/StatusCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Grid, Link, Text } from "@chakra-ui/react"
+import React from "react"
+
+function StatusCard({
+  name,
+  error,
+  revision,
+  repoURL
+}: Readonly<{
+  name: string
+  error: string
+  revision: string
+  repoURL: string
+}>): React.ReactElement {
+  return (
+    <Box borderWidth="1px" borderRadius="sm" overflow="hidden" p={4} boxShadow="lg">
+      <Grid templateColumns="auto 1fr" gap={2}>
+        <Text fontWeight="bold">Name:</Text>
```

Comment:
NIT: we have a bunch of <Text> components with the single styling fontWeight "bold". Could be extracted into a tiny component as it might be cumbersome later to change this style, which shows up throughout the UI

Reply:
Done